### PR TITLE
adding support for networking tests on the Azure platform

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
adding e2e CI testing for ovn-kubernetes to the openshift/ovn-kubernetes and openshift/cluster-network-operator repositories.

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e-azure-ovn/openshift-e2e-azure-ovn-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+workflow:
```
Comment (abhinavdahiya) [inline ci-operator/step-registry/openshift/e2e-azure-ovn/openshift-e2e-azure-ovn-workflow.yaml:1]: look at https://github.com/openshift/release/pull/9612, i think 9612 is much better way to organize these steps. I would try to use that organization.

File: ci-operator/step-registry/openshift/e2e-azure-ovn/OWNERS [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (abhinavdahiya) [inline ci-operator/step-registry/openshift/e2e-azure-ovn/OWNERS:1]: symlink maybe? like https://github.com/openshift/release/pull/9612/files#diff-fc6ab7f20b1ec73a43c9888781c761d7

**Comment by JacobTanenbaum** [comment]
ovn-kube successfully started tests: https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/9599/rehearse-9599-pull-ci-openshift-ovn-kubernetes-master-e2e-azure/1273610324892717056

cno timed out: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/9599/rehearse-9599-pull-ci-openshift-cluster-network-operator-master-e2e-azure/1273610324649447424

/retest

**Pull request merged**